service/operator/framework.go

```diff
@@ -3,6 +3,7 @@ package operator
 import (
 	"github.com/giantswarm/azure-operator/service/cloudconfig"
 	"github.com/giantswarm/azure-operator/service/resource/deployment"
+	"github.com/giantswarm/azure-operator/service/resource/dnsrecord"
```

Comment:
I would like to start versioning this by name as well. Should be `dnsrecordv1` then and the package directive and the resource name should reflect this as well.

Reply:
I'm not yet fluent with package versioning scheme we have. I want to leave that for later and change all packages at once.